lib/src/config_util.cc

```diff
@@ -0,0 +1,29 @@
+#include <internal/config_util.hpp>
+#include <cctype>
+
+namespace hocon {
+
+    bool is_whitespace(char c) {
+        switch (c) {
+            case ' ':
+            case '\n':
+            // TODO: these complain about char overflow, what do.
```

Comment:
It might be simpler to build this around regex.

I'm ok recording this as a data point to fix up later, as we get a better sense of what we need to do for unicode in the project. I'd file a ticket to track it.